### PR TITLE
15 fix popups

### PR DESCRIPTION
**Pull request by SebuBergman**

### CODE REVIEW

File: src/components/pages/adminPage/DeleteUser.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+              }}
+            >
+              Do you want to delete this user?
+            </Typography>
+          </Stack>
+          <Stack direction="row" spacing={2} justifyContent="center">
+            <Button
+              sx={confirmButton}
+              variant="contained"
+              onClick={async () => {
```
Comment (amarukorp) [inline src/components/pages/adminPage/DeleteUser.tsx:47]: This async function should be define outside the return to improve code readability 😇

File: src/components/pages/userPage/ReturnBook.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+              }}
+            >
+              Do you want to return this book?
+            </Typography>
+          </Stack>
+          <Stack direction="row" spacing={2} justifyContent="center">
+            <Button
+              sx={confirmButton}
+              variant="contained"
+              onClick={async () => {
```
Comment (amarukorp) [inline src/components/pages/userPage/ReturnBook.tsx:49]: function should be define outside the return for code readability 😇

File: src/components/pages/userPage/UserPage.tsx [hunk 3]
```diff
@@ -185,22 +188,8 @@ const MyAccount: FC = (): JSX.Element => {
                 sx={userPageReturnButton}
                 variant="contained"
                 onClick={async () => {
```
Comment (amarukorp) [inline src/components/pages/userPage/UserPage.tsx:190]: Let's make this a function as well

File: src/sxStyles.ts [hunk 4]
```diff
@@ -401,3 +401,91 @@ export const editBookListCancelButton: SxProps = {
...
+  lineHeight: 20,
+  right: -10,
+  top: -10,
+  fontSize: 24,
+  background: "#ffffff",
+  borderRadius: 18,
+  border: "1px solid #cfcece",
+};
+
+export const cancelButton: SxProps = {
```
Comment (amarukorp) [inline src/sxStyles.ts:444]: we can remove the comments from the styling if the are not needed 😇

File: .env [hunk 5]
```diff
@@ -1 +1 @@
-REACT_APP_BACKEND_URL=http://localhost:3002
```
Comment (amarukorp) [inline .env:1]: .env files should not be pushed to github you must check you gitignore file

**Review by amarukorp — CHANGES_REQUESTED** [review]
Some minor changes here and there. Overall great work 💪🏽

File: .env (hunk 5 above)
Comment (SebuBergman) [reply .env:1]: This .env file is in all the other branches. And also the change is identical to the other branches so it's kinda useless that github desided to show this

File: src/ToastContainers.tsx [hunk 6]
```diff
@@ -14,7 +14,7 @@ const ToastContainers: FC = (): JSX.Element => {
         pauseOnFocusLoss
         draggable
         pauseOnHover
-        theme="light"
+        theme="dark"
```
Comment (amarukorp) [inline src/ToastContainers.tsx:17]: why did you change to dark theme? was that show to the PO? 🤔

Comment (SebuBergman) [reply src/ToastContainers.tsx:17]: I thought it would work better with the other popups that are not toast popups. We probably should make them all similiar because some use mui alerts still i think

Comment (amarukorp) [reply src/ToastContainers.tsx:17]: Also there are some conflicting files

File: src/components/pages/adminPage/BooksGrid.tsx [hunk 7]
```diff
@@ -9,13 +9,26 @@ const BooksGrid: FC = (): JSX.Element => {
...
-    { field: "title", headerName: "Title", flex: 5, minWidth: 140 },
-    { field: "author", headerName: "Author", flex: 4, minWidth: 140 },
-    { field: "year", headerName: "Year", flex: 4, minWidth: 60 },
-    { field: "topic", headerName: "Topic", flex: 4, minWidth: 100 },
-    { field: "location", headerName: "Location", flex: 3, minWidth: 140 },
-    { field: "isbn", headerName: "ISBN", flex: 3, minWidth: 160 },
+    { field: "id", headerName: "ID", flex: 1 },
+    { field: "title", headerName: "Title", flex: 5 },
+    { field: "author", headerName: "Author", flex: 4 },
+    { field: "year", headerName: "Year", flex: 4 },
```
Comment (amarukorp) [inline src/components/pages/adminPage/BooksGrid.tsx]: I assume this changes came from the conflicting files

File: src/components/pages/adminPage/EditUsers.tsx [hunk 8]
```diff
@@ -30,6 +38,14 @@ const EditUser: FC<IProps> = ({
 }: IProps): JSX.Element => {
   const editingMessage = () =>
     toast.success("User edited succesfully", { containerId: "ToastSuccess" });
+  const [offices, setOffices] = useState<HomeOffice[]>([]);
```
Comment (amarukorp) [inline src/components/pages/adminPage/EditUsers.tsx:37]: same here?

File: src/components/pages/listBooksPage/ButtonPopup.tsx [hunk 9]
```diff
@@ -2,6 +2,7 @@ import { FC } from "react";
 import { Modal, Box, Button, Stack, Typography } from "@mui/material";
 import { popupContainer, confirmButton, cancelButton } from "../../../sxStyles";
 import { toast } from "react-toastify";
+
 import "react-toastify/dist/ReactToastify.css";
```
Comment (amarukorp) [inline src/components/pages/listBooksPage/ButtonPopup.tsx:7]: All good here

**Review by amarukorp — APPROVED** [review]
Seems good to me!
